### ISSUE TITLE
Serialized term meta values are discarded on import

### ISSUE THREAD

**Issue by ChrisHardie** [WordPress/wordpress-importer#12]
Today I noticed that a WXR export file with terms having serialized values was importing without obvious error, but that said values were being discarded from the imported term. 

Example WXR line:

```
<wp:term><wp:term_id><![CDATA[1]]></wp:term_id><wp:term_taxonomy><![CDATA[travelguide]]></wp:term_taxonomy><wp:term_slug><![CDATA[new_york_city]]></wp:term_slug><wp:term_parent><![CDATA[north-america]]></wp:term_parent><wp:term_name><![CDATA[New York]]></wp:term_name><wp:term_description><![CDATA[]]></wp:term_description><wp:termmeta><wp:meta_key><![CDATA[travelguide_info]]></wp:meta_key><wp:meta_value><![CDATA[a:3:{s:8:"latitude";s:7:"40.7256";s:9:"longitude";s:10:"-73.994564";s:8:"keywords";s:68:"New York,Brooklyn,Bronx,Queens,Staten Island,The Big Apple,Manhattan";}]]></wp:meta_value></wp:termmeta></wp:term>
```

The resulting row in `wp_termmeta` would be something like:

```
+---------+---------+------------------+------------+
| meta_id | term_id | meta_key         | meta_value |
+---------+---------+------------------+------------+
|     219 |   19096 | travelguide_info |            |
+---------+---------+------------------+------------+
```

I believe this is happening because of the call to `wp_slash` at https://github.com/boonebgorges/wordpress-importer/blob/master/wordpress-importer.php#L507 which leads to an empty value after `maybe_unserialize` at https://github.com/boonebgorges/wordpress-importer/blob/master/wordpress-importer.php#L573
Terms with term meta values that are not serialized import just fine.

Happy to provide more details or detailed reproduction steps if needed.

**Comment by boonebgorges** [comment]
Thanks, @ChrisHardie. I assume we're passing through `wp_slash()` because the term functions expect slashed values, but we forgot that `termmeta` part of the term array. A simple workaround is probably just to store the value of `$term['termmeta']` in a variable before running through `wp_slash()`, and then to add it back to `$term` after the slashing has taken place.

If you get a chance to put that theory to a test and create a PR, I'd be grateful :)

**Comment by ChrisHardie** [comment]
Submitted a PR, which fixed my import cases. Let me know if I can be helpful in further testing/refinement.

**Comment by ChrisHardie** [comment]
I'm now not sure this PR completely addresses the issue. In my testing I was importing a WXR file with just the terms, but today when I did an import that had the terms plus content items that referenced said terms, the termmeta was deleted, serialized or not. (Strangely, it only happened for 100+ content items, not 75 (for example). It also didn't happen if I imported the terms and content separately.)

I can't be sure that this issue is related to the one I initially identified, but I know I can't say that serialized termmeta importing is working just yet. Will keep looking.
